mbf_abstract_nav/include/mbf_abstract_nav/abstract_action_base.hpp

```diff
@@ -131,28 +132,31 @@ class AbstractActionBase
   {
     uint8_t slot = goal_handle.getGoal()->concurrency_slot;
 
-    if(goal_handle.getGoalStatus().status == actionlib_msgs::GoalStatus::RECALLING)
+    if (goal_handle.getGoalStatus().status == actionlib_msgs::GoalStatus::RECALLING)
     {
       goal_handle.setCanceled();
     }
     else
     {
-      boost::lock_guard<boost::mutex> guard(slot_map_mtx_);
+      slot_map_mtx_.lock();
       typename ConcurrencyMap::iterator slot_it = concurrency_slots_.find(slot);
-      if (slot_it != concurrency_slots_.end() && slot_it->second.in_use) {
+      slot_map_mtx_.unlock();
+      if (slot_it != concurrency_slots_.end() && slot_it->second.in_use)
+      {
         // if there is already a plugin running on the same slot, cancel it
         slot_it->second.execution->cancel();
 
-        if (slot_it->second.thread_ptr->joinable()) {
+        if (slot_it->second.thread_ptr->joinable())
```

Comment:
not sure why you formatted this. did you run `git clang-format master`?

Reply:
The idea was to format little by little with the PRs, right?
I can undo, anyway

Reply:
sure, but not at random :grin: